src/ray/core_worker/test/core_worker_test.cc

```diff
@@ -88,7 +88,7 @@ std::string MetadataToString(std::shared_ptr<RayObject> obj) {
   return std::string(reinterpret_cast<const char *>(metadata->Data()), metadata->Size());
 }
 
-class CoreWorkerTest : public ::testing::Test {
+class CoreWorkerTest : public RedisServiceManagerForTest {
```

Comment:
Could you add a comment here on why it should inherit from `RedisServiceManagerForTest`? e.g. for setting up redis server .etc. Maybe this can be changed to composition instead of inheritance.

Reply:
done